workspaces/core/src/select/Select.tsx

```diff
@@ -0,0 +1,152 @@
+import React, {useEffect, useRef, useState} from "react";
+import ChevronDown from "../private/icons/ChrvronDown";
+import ChevronUp from "../private/icons/ChrvronUp";
+import Option from "./private/types/OptionProps";
+import SelectProps from "./private/types/SelectProps";
+
+const Select: React.FC<SelectProps> = ({
+   options,
+   label,
+   value,
+   disabled,
+   isError,
+   placeholderSlot,
+   placeholderValue,
+   hintSlot,
+   onChange,
+   size = 'md',
+   amountOfVisibleItems = 9999
+}) => {
+  const [menuOpen, setMenuOpen] = useState(false);
+  const [activeOption, setActiveOption] = useState(0);
+  const inputRef = useRef(null);
+  const selectRef = useRef(null);
+  const menuRef = useRef(null);
+  const placeholder = value ? options?.filter((option) => option.value === value)[0]?.label
+    : placeholderSlot ?? placeholderValue ?? 'Choose an option';
+  let classNames = `flex relative py-${ size === 'md' ? 2 : 3 } px-${ size === 'md' ? 3 : 4 } bg-gohan rounded-lg border border-solid`;
```

Comment:
Please don't use string concatenation to create class names. Please rewrite like this: 

```
${ size === 'md' ? 'py-2' : 'py-3' }
```